magento/magento2
# fixed issue of Backup tool not correctly detecting .maintenance.flag

Fixed issue of Backup tool not correctly detecting .maintenance.flag

### Description (*)
This PR fix the issue of backup tool not correct detecting .maintenance.flag issue in Magento 2.3.0. 

### Fixed Issues (if relevant)
1. magento/magento2#19825: Magento 2.3.0: Backup tool not correctly detecting .maintenance.flag

### Manual testing scenarios (*)
Case 1:
1. Go to Dashboard -> System -> Tools -> Backups -> Database Backup
2. Enter backup name, check "Maintenance mode", click OK.
3. Back up is created 

Case 2: 

1. Go to Dashboard -> System -> Tools -> Backups -> Database Backup
2. Enter backup name, and click OK.
3. Back up is created.


### Contribution checklist (*)
